components/script/dom/webglrenderingcontext.rs

```diff
@@ -2357,24 +2357,32 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     }
 
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.9
-    fn GetShaderPrecisionFormat(&self,
-                                shader_type: u32,
-                                precision_type: u32)
-                                -> Option<DomRoot<WebGLShaderPrecisionFormat>> {
+    fn GetShaderPrecisionFormat(
+        &self,
+        shader_type: u32,
+        precision_type: u32
+    ) -> Option<DomRoot<WebGLShaderPrecisionFormat>> {
+        let precision_type_matches = match precision_type {
+            constants::LOW_FLOAT |
+            constants::MEDIUM_FLOAT |
+            constants::HIGH_FLOAT |
+            constants::LOW_INT |
+            constants::MEDIUM_INT |
+            constants::HIGH_INT => true,
```

Comment:
For a single validation that doesn't duplicate code, I'd remove the variable entirely and do:

```rust
match precision_type {
    FOO | BAR | BAZ => {},
    _ => {
        self.webgl_error(InvalidEnum);
        return None;
    }
}
```
wdyt?

Reply:
Yeah, I was thinking of that too. Let's do it this way then.